lib/frontend/constructs/hosting.ts

```diff
@@ -0,0 +1,20 @@
+import * as cdk from "@aws-cdk/core";
+import * as cloudfront from "@aws-cdk/aws-cloudfront";
+import * as origins from "@aws-cdk/aws-cloudfront-origins";
+import * as s3 from "@aws-cdk/aws-s3";
+
+export class Hosting extends cdk.Construct {
```

Comment:
This class is specifally tight to the frontend stuff, right?

Do we than need to pass in the `id` as a parameter, whereas other settings like bucket name and cloudfront distribution name are hardcoded? 🤔